src/pages/Bets/components/BetRows.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { BetHistory } from "../../../types";
+import { Row } from "./BetTable";
+
+type Props = {
+  bets?: BetHistory[];
+  onClickBet: (bet?: BetHistory) => void;
+};
+
+const BetRows: React.FC<Props> = ({ bets, onClickBet }) => (
+  <React.Fragment>
+    {!bets ? (
```

Comment:
should be bets.length===0?

Reply:
`bets` is undefined while the data loads from the api